frontend/components/ArticleBody.tsx

```diff
@@ -577,7 +583,7 @@ const ArticleBody: React.SFC<{
                         </div>
                     )}
                     <div className={dateline}>
-                        {dtFormat(CAPI.webPublicationDate)}
+                        {dtFormatGU(CAPI.webPublicationDate, edition)}
                     </div>
```

Comment:
Would it be possible to extract this into a dateline component, and move the cookie-checking logic there? This will help keep `Article` stateless, and will save us from re-rendering the entire article after it has mounted.

Reply:
Good idea! On it.